### DISCUSSION TITLE
Smartwings blinds no battery reporting

### DISCUSSION THREAD

**Discussion by fernv77** [Koenkk/zigbee2mqtt#18050] [Devices]
Good Afternoon,

I have some rollershades from Smartwings that are Homekit compatible. I recently ordered a Terncy hub which I saw from a channel on Youtube stating you can flash them to ZIgbee with it. I did so because I was experiencing slow response and sometimes getting "not responding" from Siri with the shades. I flashed one and was able to add to ZIgbee2MQTT using an external converter. Thing is they do not report the battery percentage. The device that shows full compatibility on the site is WM25L-Z while my device name is showing on Z2M as Terncy-RM03. I know that the WM25L-Z is originally Zigbee. Checking the Clusters I do not see genPowerCfg and logs show it's an unsupported attribute. Ive held off from flashing the rest to Zigbee until I can solve this. I cant even flash them back to the Homekit firmware. I found that out after I had flashed the firmware. Since I am new to this what would you guys need to know or see from my set up so we can get the battery percentage showing.  

Thanks!

**Comment by pfremm** [comment]
Did you ever figure this out?  I saw battery on ZHA but I now I don't with Z2M.

**Reply by fernv77** [reply]
Unfortunately I did not. Someone on a forum (don’t remember which one) confirmed they won’t show battery percentage. I just ordered a a real Zigbee replacement motor and now it works fine.

**Reply by pfremm** [reply]
I definitely had battery levels on ZHA so seems like it must be a configuration issue.
